easybuild/easyconfigs/k/kineto/kineto-0.4.0-GCC-11.3.0.eb

```
@@ -0,0 +1,34 @@
+easyblock = 'CMakeMake'
+
+name = 'kineto'
+version = '0.4.0'
+
+homepage = 'https://github.com/pytorch/kineto'
+description = "A CPU+GPU Profiling library that provides access to timeline traces and hardware performance counters"
+
+toolchain = {'name': 'GCC', 'version': '11.3.0'}
+
+source_urls = ['https://github.com/pytorch/kineto/archive/']
+sources = [{
+    'git_config': {
+        'url': 'https://github.com/pytorch',
+        'repo_name': name,
+        'tag': 'v%(version)s',
+        'recursive': True,
+    },
+    'filename': SOURCE_TAR_GZ,
+}]
+checksums = [None]
+
+builddependencies = [
+    ('CMake', '3.24.3'),
+]
```

```suggestion
    ('Python', '3.10.4', '-bare'),
]
```

From the buildlog
```
-- Found PythonInterp: /rds/projects/2017/branfosj-rse/easybuild/venv-dev-Ubuntu20.04/bin/python (found version "3.8.5")
```

And we see that used at https://github.com/pytorch/kineto/blob/main/libkineto/CMakeLists.txt#L14